worth2/templates/main/avatar_selector_block.html

```diff
@@ -13,15 +7,13 @@
 <form class="worth-avatars" method="post">
```

Comment:
This is where some of the pain might be. page.html creates a form that is used by all blocks, and feeds into PageView.submit. This block template is setting up a nested form which is not supported. (http://www.w3.org/TR/2011/WD-html5-20110525/forms.html#the-form-element) Some browsers, like Chrome, go so far as to strip the nested form tags out of the html.